types/web3/index.d.ts

```diff
@@ -53,5 +53,3 @@ declare class Web3 {
     setProvider(provider: Provider): void;
     utils: Utils;
 }
-
-export = Web3;
```

Comment:
this is compatible only with web3.js 1.x

https://github.com/ethereum/web3.js/blob/1.x/packages/web3/src/index.js

Reply:
now I am using like this `import web3 = require('web3')`

`new web3.Web3('ws://localhost:8545')`

I am getting

`TypeError: web3.Web3 is not a constructor`

can u plz help me how to solve this!

Reply:
```typescript
import Web3 = require('web3')
const web3 = new Web3()
```